clerk/organizations.go

```diff
@@ -296,3 +296,18 @@ func (s *OrganizationsService) UpdateMembership(organizationID string, params Up
 	}
 	return &organizationMembership, nil
 }
+
+type DeleteOrganizationMembershipParams struct {
+	UserID string
+}
+
+func (s *OrganizationsService) DeleteMembership(organizationID string, params DeleteOrganizationMembershipParams) (*OrganizationMembership, error) {
```

Comment:
🔧 Instead of passing a struct here, we must always accept userID as a parameter
```suggestion
func (s *OrganizationsService) DeleteMembership(organizationID, userID string) (*OrganizationMembership, error) {
```

Reply:
@chanioxaris  Code has been update! 💯